src/server/shared/DataStores/DBCStructure.h

```diff
@@ -672,25 +672,23 @@ struct ChrRacesEntry
     uint32      expansion;                                  // 68 (0 - original race, 1 - tbc addon, ...)
 };
 
-/* not used
 struct CinematicCameraEntry
 {
     uint32      id;                                         // 0 index
     char*       filename;                                   // 1
     uint32      soundid;                                    // 2 in SoundEntries.dbc or 0
-    float       start_x;                                    // 3
-    float       start_y;                                    // 4
-    float       start_z;                                    // 5
-    float       unk6;                                       // 6 speed?
+    float       base_x;                                     // 3
+    float       base_y;                                     // 4
+    float       base_z;                                     // 5
+    float       base_o;                                     // 6
 };
-*/
 
 struct CinematicSequencesEntry
 {
     uint32      Id;                                         // 0 index
     //uint32      unk1;                                     // 1 always 0
-    //uint32      cinematicCamera;                          // 2 id in CinematicCamera.dbc
-    // 3-9 always 0
+    uint32      cinematicCamera;                            // 2 id in CinematicCamera.dbc
+                                                            // 3-9 always 0
```

Comment:
?

Reply:
?

Reply:
Why add two spaces on line 690 instead of using tabs?